includes/defaults.inc.php

```diff
@@ -712,6 +712,7 @@
 $config['poller_modules']['ntp']                         = 1;
 $config['poller_modules']['services']                    = 1;
 $config['poller_modules']['loadbalancers']               = 0;
+$config['poller_modules']['mef-evc']                     = 0;
```

Comment:
It's a question rather than an ask. I just don't want to see us end up with 10 modules all called mef-this, mef-that, etc.